pkg/controllers/bigtableautoscaler_controller.go

```diff
@@ -125,6 +118,11 @@ func (r *BigtableAutoscalerReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		autoscaler.Status.CurrentCPUUtilization = &cpuUsage
 	}
 
+	if autoscaler.Status.CurrentNodes == nil {
```

Comment:
Since this is a pointer its initial value is `nil`, that is why I need to create a variable here.
We could refactor the spec to be a value instead of a pointer, though I think there is a coming PR that will be better to do this.